Taskfile.yml

```diff
@@ -141,6 +144,7 @@ tasks:
       - "{{.SRC_DIR}}/CMakeLists.txt"
       - "{{.SRC_DIR}}/src/**/*"
       - "{{.TASKFILE_DIR}}/Taskfile.yml"
+      - "/etc/os-release"
```

Comment:
That's right. Or it could be us who use one container to build and publish a package first, and later use another container with a different OS to build and publish another package.